src/cli/commands/migrate.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+import { type ChangelogAPIType } from '../../types';
+
+export default async function runMigrate(
```

Comment:
Generic thought: did you know about umzug js? Just wondering whether logic involved in the process of migration should reside in strangelog itself :-)